tests/manual/topologies/README.md

```diff
@@ -0,0 +1,24 @@
+## Background
+Below are a couple of scenarios to test deployment topologies introduced in `INTEGRATING.md`.
+
+
+### Relating directly to cos-proxy
+- Deploy [`cos-lite`](https://charmhub.io/topics/canonical-observability-stack) in a k8s model
+  - Offer `grafana:grafana-dashboard` and `prometheus:metrics-endpoint` endpoints
+- Consume `grafana:grafana-dashboard` and `prometheus:metrics-endpoint` in a newly created lxd model 
```

Comment:
The bundle already has a saas section so iirc, the manual consume step is not necessary.